homeassistant/components/demo/geo_location.py

```diff
@@ -152,7 +153,7 @@ def longitude(self) -> float | None:
         """Return longitude value of this external event."""
         return self._longitude
 
-    @property
+    @cached_property
     def unit_of_measurement(self) -> str:
```

Comment:
If it's this static, shouldn't we just set it to `_attr_unit_of_measurement` ? Oh I guess that still goes through the method and so a cached property is slower.